### PR TITLE
Set github token from env

### LINKED ISSUE #64: GitHub comment fails on private repos
Travis.com allows tests to be run on private GitHub repos. However, the comments fail because the comment_token doesn't have access to the private repo. Here's the error message:

<pre><code>Error in stop(httr::http_condition(response, "error", message = httr::content(response,  : 
  {
  "message": "Not Found",
  "documentation_url": "https://developer.github.com/v3"
}
Calls: print -> print.lints -> github_comment
</code></pre>

For our purposes, it'd be nice to simply turn off GitHub commenting entirely. However, it might also be a good idea to have this fail a bit more gracefully and not halt the program. When this happens, we don't see any of the lint results in our Travis CI logs.

### PR DESCRIPTION
**Pull request by mattyb**
Fixes #64

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
 
 language: R
+r:
+  - 3.4
```
Comment (jimhester) [inline .travis.yml]: I would prefer to change the travis yaml independent of the other changes.

**Comment by jimhester** [comment]
Thanks! Overall this looks good, can you add a note to NEWS.md describing the change and mentioning the issue number and your GitHub username, then I can merge this.

File: .travis.yml (hunk 1 above)
Comment (mattyb) [reply .travis.yml]: makes sense! I was trying to debug #363. Will revert.

**Comment by mattyb** [comment]
updated!

**Comment by jimhester** [comment]
Thanks!

**Pull request merged**